### ISSUE TITLE
Unable to  disable default http backend

### ISSUE THREAD

**Issue by abhi1693** [rancher/terraform-provider-rke#311]
As per https://rancher.com/docs/rke/latest/en/config-options/add-ons/ingress-controllers/#disabling-nginx-ingress-default-backend, I should be able to disable the default backend. However, I get the following error

```
╷
│ Error: Unsupported argument
│ 
│   on ../../modules/rke/main.tf line 39, in resource "rke_cluster" "cluster":
│   39:     default_backend = false
│ 
│ An argument named "default_backend" is not expected here.

```

Code snippet:
```
ingress {
    provider = "nginx"
    default_backend = false
    options = {
      server_tokens = false
    }
  }
```

**Comment by rawmind0** [comment]
As per https://registry.terraform.io/providers/rancher/rke/latest/docs/resources/cluster#ingress, `default_backend` argument is not yet supported.

**Comment by abhi1693** [comment]
@rawmind0 I opened a PR to add support. Is there something wrong with my implementation?

**Comment by rawmind0** [comment]
> @rawmind0 I opened a PR to add support. Is there something wrong with my implementation?

yes, https://github.com/rancher/terraform-provider-rke/pull/312#pullrequestreview-765469458 . Please, at least try to build and test the provider before submit the PR, `make` and `make test`
